README.md

```diff
@@ -100,23 +100,24 @@ Aleth supports EVMC out of the box, but geth not yet.
 
 1. Build geth with EVMC
 
-	Checkout the `evmc` branch of [this geth fork](https://github.com/chfast/go-ethereum/tree/evmc) ([PR](https://github.com/ethereum/go-ethereum/pull/17050)):
-	
+	Clone the [the ewasm geth fork](https://github.com/ewasm/go-ethereum):
+
+	Geth for ewasm testnet is released regularly by tagging the above fork with `ewasm-testnet-milestoneX`, where
+	`X` stands for a milestone number.
+
+	(Latest: [`ewasm-testnet-milestone3`](https://github.com/ewasm/go-ethereum/tree/ewasm-testnet-milestone3))
+
 	```sh
-	> git clone https://github.com/chfast/go-ethereum && cd go-ethereum
-	> git checkout evmc
+	> git clone https://github.com/ewasm/go-ethereum -b ewasm-testnet-milestone3
```

Comment:
So is it milestone3 or milestone1?

Reply:
Up to you, for some reason I thought I heard 3 on the call

Reply:
Okay changed to 1 for now.